src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +22,13 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(s -> s.split(FIRST_TASK_SPLIT_SEPARATOR))
+                .flatMap(Arrays::stream)
+                .mapToInt(Integer::valueOf)
```

Comment:
`valueOf` returns wrapper, `parseInt` - primitive 
```suggestion
                .mapToInt(Integer::parseInt)
```